### PR TITLE
update used gh actions ahead of node12 deprecation

### PR DESCRIPTION
**Pull request by davidbloss**
Please merge this update today! [node 12 support dropping tomorrow](https://github.blog/changelog/2023-05-04-github-actions-all-actions-will-run-on-node16-instead-of-node12/) and these version updates to Github Actions get ahead of that. Resolves RELENG-144

### CODE REVIEW

File: website/blog/2022-07-26-pre-commit-dbt.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ jobs:
     # This action will output all the files that are being created and modified in our PR
       - name: Get changed files
         id: get_file_changes
-        uses: trilom/file-changes-action@v1.2.4
+        uses: trilom/file-changes-action@v1
```
Comment (runleonarun) [inline website/blog/2022-07-26-pre-commit-dbt.md]: Looks like 1.2.4 is their latest: https://github.com/marketplace/actions/file-changes-action
```suggestion
        uses: trilom/file-changes-action@v1.2.4
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
Adding one change then we can merge!

**Review by runleonarun — DISMISSED** [review]
LGTM

**Comment by runleonarun** [comment]
@JKarlavige Does this PR look OK to you to merge?

**Review by JKarlavige — DISMISSED** [review]
This lgtm. All the checks still ran and passed within this PR so no concerns here!

**Review by JKarlavige — APPROVED** [review]
🚢

**Pull request merged**